tests/check_ghostty_zig_workflows.py

```
@@ -1,60 +1,436 @@
+        for part in getattr(command, "parts", ())
+        if getattr(part, "kind", None) == "word"
+    ]
+
+
+def _skip_options(words: list[str], index: int) -> int:
+    while index < len(words) and words[index].startswith("-"):
+        index += 1
+    return index
+
+
+def _executed_target(words: list[str]) -> str | None:
```

Ruff (0.16.0)

[warning] Too many return statements (7 > 6)

(PLR0911)